src/main/java/com/wolox/training/models/User.java

```diff
@@ -31,7 +31,7 @@
   @Column(nullable = false)
   private LocalDate birthDate;
 
-  @OneToMany(mappedBy = "user", cascade = CascadeType.ALL, orphanRemoval = true)
+  @OneToMany(mappedBy = "user", cascade = CascadeType.PERSIST)
```

Comment:
The problem here is that if you want to save an user that already exists just because you change for example his name, then the books will be persisted again. And that is not what we are looking for, we just need MERGE and REFRESH

You can look at MERGE as the instruction to "merge" changes on the model back to the database.

But we're lacking the one that brings modifications in the database to the model. And that's REFRESH :)